netweaver/monitoring.sls

```diff
@@ -0,0 +1,30 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+
+prometheus_sap_host_exporter_pkg:
+  pkg.installed:
+    - name: prometheus-sap_host_exporter
+
+sap_host_exporter_exporter_service:
+  service.running:
+    - name: prometheus-sap_host_exporter
+    - enable: True
+    - restart: True
+    - require:
+      - pkg: prometheus_sap_host_exporter_pkg
+      - file: sap_host_exporter_configuration
+
+# for each node the instance number change so we need to adapt configuration
+{% for node in netweaver.nodes if host == node.host %}
+sap_host_exporter_configuration:
+  file.managed:
+    - name: /etc/sap_host_exporter.yaml
+    - contents: |
+         port: "9680"
+         addres: "0.0.0.0"
```

Comment:
It should be `addres**s**`, right? (double `s` at the end)

Reply:
https://github.com/SUSE/sap_host_exporter/pull/12
yep